manager/qemu/config.go

```diff
@@ -70,8 +72,12 @@ type Config struct {
 	CPU      string `env:"CPU" envDefault:"EPYC"`
 	SmpCount int    `env:"SMP_COUNT" envDefault:"4"`
 	MaxCpus  int    `env:"SMP_MAXCPUS" envDefault:"64"`
+	MEM_ID   string `env:"MEM_ID" envDefault:"ram1"`
```

Comment:
Rename this to `MemID`, Go idiomatic way of naming vars

Reply:
My mistake, I have changed the name now.